### PR TITLE
dev-java/asm: slot 3 again

### PR DESCRIPTION
**Pull request by vaukai**

### CODE REVIEW

File: dev-java/asm/asm-3.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+DEPEND=">=virtual/jdk-1.8
+	test? (
+		dev-java/aspectj:0
+		dev-java/bcel:0
+		dev-java/cojen:0
+		dev-java/janino:0
+		dev-java/javassist:3
+		dev-java/jclasslib:0
+		dev-java/junit:0
```
Comment (fordfrog) [inline dev-java/asm/asm-3.3.2.ebuild:41]: you have `junit:0` here but `junit-4` in `JAVA_TESTING_FRAMEWORKS`. i suppose the slots should be the same.

Comment (vaukai) [reply dev-java/asm/asm-3.3.2.ebuild]: > you have `junit:0` here but `junit-4` in `JAVA_TESTING_FRAMEWORKS`. i suppose the slots should be the same.

Adjusted. Still missing some dependencies:
```
 package alt.jiapi.reflect does not exist
 package com.claritysys.jvm.builder does not exist
 package com.claritysys.jvm.classfile does not exist
 package gnu.bytecode does not exist
 package gnu.bytecode.Type does not exist
 package jbet does not exist
 package org.mozilla.classfile does not exist
 package serp.bytecode does not exist
```
Any idea where to find `gnu.bytecode`or `com.claritysys.jvm`?

**Comment by vaukai** [comment]
```
# grep 'does not exist' /var/tmp/portage/dev-java/asm-3.3.2/temp/build.log | cut -d':' -f4 | sort | uniq
# package gnu.bytecode does not exist
# package gnu.bytecode.Type does not exist
# package org.mozilla.classfile does not exist
``` 
```
# those bundled jar files needed for testing seem not available from anywhere else
JAVA_GENTOO_CLASSPATH_EXTRA="test/lib/jbet3-R1.jar:test/lib/jiapi.jar:test/lib/csg-bytecode.jar:test/lib/serp-1.14.2.jar"
```
See https://gitlab.ow2.org/asm/asm/-/tree/390a3ba575031f80f526c585c864324b42ee62a7/test
But still missing these:
```
# grep 'does not exist' /var/tmp/portage/dev-java/asm-3.3.2/temp/build.log | cut -d':' -f4 | sort | uniq
# package gnu.bytecode does not exist
# package gnu.bytecode.Type does not exist
# package org.mozilla.classfile does not exist
```

**Comment by vaukai** [comment]
We'd better wait for an [upstream patch](https://issues.apache.org/jira/browse/COMPRESS-582) instead of adding outdated stuff.
Closing this PR.

**Comment by fordfrog** [comment]
@vaukai ok, thank you!

**Pull request closed without merging**